### DISCUSSION TITLE
Protokoll 28.11.24

### DISCUSSION THREAD

**Discussion by jac0mp** [AuDojo/AuDojo#76] [Protokolle]
# Meeting 28.11.24

### Mergesort
- große Merge schritte könnten besser visualisiert werden (kein muss, wäre aber besser)
    - box um Partitionen
    - mergesort macht kopien der arrays (aktuell sieht es nach inplace algortihmus aus)
- splitten interaktiv druchs klicken gestalten

### Bubble Sort
- user interface so gestalten, dass man zu tauschende Punkte anklickt
- verwende Visualisierungen / interaktionen zum animieren statt neue tabelleneinträge
-> Elemente druch klicks vertauschen

### Quick Sort
- pivot-Element selbst anklicken und zurückgeben (inplace arbeiten)

### Minseos Hilfeseite
- informelle description für Grundidee (Wikipedia etc.)

### Impressum & Datenschutz
- Institutsadresse (Head of institute)
- keine Texte & Bilder aus dem Internet
- Logo gehört uns?
- generell etwas aufpassen mit Bildern

## Ausblick
- Trang & An primär frontend
- backend & login

## Fragen & Antworten und allgemeines zum Deployment
- wann kann server gehostet werden?
    - wenn dockerisiert
- Requests vom Frontend limitieren

Struktur ist etwas wild, da ich erst mitten im Meeting angefangen habe

**Comment by anomatomato** [comment]
:100:
